app/assets/stylesheets/components/_index.scss

```diff
@@ -9,3 +9,8 @@
 @import "user";
 @import "select2";
 @import "recipeindex";
+@import "footer";
+
+a {
+  color: $red;
```

Comment:
just checking if this is in the proper file?

Reply:
wasnt exactly sure but that did the trick (wanted it to apply to all pages) so i just left it there